lib/typescript/render/providers/facebook/FacebookRender.tsx

```diff
@@ -74,17 +96,40 @@ function facebookInbound(message: Message): ContentUnion {
 
 function facebookOutbound(message: Message): ContentUnion {
   const messageJson = JSON.parse(message.content);
-  if (messageJson.attachment) {
-    return parseAttachment(messageJson.attachment);
-  } else if (messageJson.text) {
+
+  if (messageJson.quick_replies) {
+    if (messageJson.quick_replies.length > 13) {
```

Comment:
Facebook QuickReplies support up to 13 buttons so I added a check so that we only display 13 max.